app/(gcforms)/[locale]/(form administration)/form-builder/[id]/settings/components/ResponseDelivery.tsx

```diff
@@ -33,6 +33,12 @@ enum DeliveryOption {
   email = "email",
 }
 
+export enum PurposeOption {
```

Comment:
Thinking we should start commenting these to make it easier to folks looking at these types

i.e. something like 


```
/*
  * PurposeOption is used to determine the purpose of the form
  * admin: The form is used to collect personal information
  * nonAdmin: The form is used to collect non-personal information
  */
```



I know these don't exist for Vault etc... but might make it easier to folks onboarding or who haven't touched this part of the code.

Reply:
Agreed.